### ISSUE TITLE
at least n-history is broken with zsh version 5.8

### ISSUE THREAD

**Issue by Rabattkarte** [psprint/zsh-navigation-tools#31]
Today I upgraded from `zsh 5.7.1` (shipped with macOS 10.15.4) to `zsh 5.8` (via Homebrew).

When I use `n-history` with `zsh 5.8` I get glitchy words whenever I scroll the text or search. Please [**see this video**](https://drive.google.com/file/d/1-XV1fO9RrPya8j7gIHs8YTv8WW2riCcd/view) for details.

How to reproduce (on macOS 10.15.4):

```zsh
[~]$ brew install zsh                # Install via Homebrew
[~]$ brew info zsh
zsh: stable 5.8 (bottled), HEAD
UNIX shell (command interpreter)

[~]$ /usr/local/bin/zsh              # Use zsh installed by brew
[~]$ /usr/local/bin/zsh --version
zsh 5.8 (x86_64-apple-darwin19.3.0)
[~]$ n-history                       # see the bug here
[~]$ /bin/zsh                        # Use zsh provided by Apple
[~]$ /bin/zsh --version
zsh 5.7.1 (x86_64-apple-darwin19.0)
[~]$ n-history                       # see NO bug here
```

Possibly other tools are affected as well.

**Comment by Rabattkarte** [comment]
Here are [the release notes of zsh](http://zsh.sourceforge.net/releases.html). Maybe that helps? Also to mitigate this issue I downgraded zsh again.

**Comment by danthegoodman1** [comment]
I have a very similar issue, see the above mention.

**Comment by danthegoodman1** [comment]
@Rabattkarte Which version did you downgrade to?

**Comment by Rabattkarte** [comment]
I downgraded to zsh 5.7.1 (shipped with macOS) from zsh 5.8 (via Homebrew).

**Comment by danthegoodman1** [comment]
Yeah I just switched my shell to that as well, seems to fix the issue! Hopefully this can be esier to find for googlers because this thread took me days to find after backtracking through the ZSH chain to get to this package.

_sometimes googling for code/tech solutions doesn't work... 😢_

**Comment by mbenatti** [comment]
I have this problem too, any news to fix it on 5.8?
